.wordlist.txt

```diff
@@ -583,6 +593,7 @@ optimizers
 os
 pageable
 parallelization
+parallelize
```

Comment:
adapt (a program) for running on a parallel processing system.
"special software parallelizes C-based programs"